provisioning/install-precice.sh

```diff
@@ -60,20 +62,7 @@ pip3 install --upgrade pip
 pip3 install --user pyprecice
 
 # Additional python packages
-pip3 install --user pandas # Needed for the post-processing script of the oscillator tutorial
-
-# Temporary workaround for https://github.com/precice/vm/issues/61
-# Remove as soon as https://github.com/precice/tutorials/issues/217 gets resolved
-sudo apt-get install -y python3.8-venv
-(
-    cd tutorials/perpendicular-flap/fluid-nutils/
-    python3 -m venv nutils6-env
-    # shellcheck source=/dev/null
-    source nutils6-env/bin/activate
-    pip3 install nutils==6.3 pyprecice
-    sed -i "s/python3/nutils6-env\/bin\/python3/g" ./run.sh
-    deactivate
-)
+pip3 install --user pandas matplotlib # Needed for the post-processing scripts
```

Comment:
polars is needed by precice-profiling

```suggestion
pip3 install --user pandas matplotlib polars # Needed for the post-processing scripts
```